### PR TITLE
Fix #101: Display "external_user_id" for blocked activations

### LINKED ISSUE #101: Display "external_user_id" for blocked activations
We should reflect changes from https://github.com/wultra/powerauth-server/issues/280 in the PowerAuth admin web interface.

### PR DESCRIPTION
**Pull request by romanstrobl**
Updated changes tab + detection of LDAP username.

### CODE REVIEW

File: powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java [hunk 1]
```diff
@@ -216,7 +223,7 @@ public String activationCreateCommitAction(@RequestParam(value = "activationId")
      */
     @RequestMapping(value = "/activation/block/do.submit", method = RequestMethod.POST)
     public String blockActivation(@RequestParam(value = "activationId") String activationId, @RequestParam(value = "redirectUserId") String userId, Map<String, Object> model) {
-        BlockActivationResponse blockActivation = client.blockActivation(activationId, null);
+        BlockActivationResponse blockActivation = client.blockActivation(activationId, null, ldapService.getLdapUsername());
```
[the remark this answers is not in the document]
Comment (petrdvorak) [reply powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java]:
```java
public String blockActivation(
    @RequestParam(value = "activationId") String activationId, 
    @RequestParam(value = "redirectUserId") String userId, 
    Map<String, Object> model, 
    Principal principal) {
    String username = principal.getName();
// ...
}
```

File: powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java [hunk 2]
```diff
@@ -233,7 +240,7 @@ public String blockActivation(@RequestParam(value = "activationId") String activ
      */
     @RequestMapping(value = "/activation/unblock/do.submit", method = RequestMethod.POST)
     public String unblockActivation(@RequestParam(value = "activationId") String activationId, @RequestParam(value = "redirectUserId") String userId, Map<String, Object> model) {
-        UnblockActivationResponse unblockActivation = client.unblockActivation(activationId);
+        UnblockActivationResponse unblockActivation = client.unblockActivation(activationId, ldapService.getLdapUsername());
```
Comment (petrdvorak) [inline powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java]: dtto

File: powerauth-admin/src/main/java/io/getlime/security/app/admin/service/LdapDetailsService.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service for obtaining LDAP authentication details.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Service
+public class LdapDetailsService {
```
Comment (petrdvorak) [inline powerauth-admin/src/main/java/io/getlime/security/app/admin/service/LdapDetailsService.java]: See above. This causes direct dependency of our authentication with LDAP. We would like to reuse any type of logged in user, regardless of specific authentication technology.

File: powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (romanstrobl) [reply powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java]: Ok, this way seems much simpler.

**Pull request merged**